src/util_i_csvlists.nss

```diff
@@ -264,6 +268,27 @@ string RemoveListItem(string sList, string sListItem)
     return DeleteListItem(sList, FindListItem(sList, sListItem));
 }
 
+int CopyListItem(string sSource, string sTarget, int nIndex, int bAddUnique = FALSE, int nRange = 1)

```

Comment:
Ha, good call. Sometimes I greet the functions of csv and varlists confused when I'm not paying attention. I'll get it fixed.

Reply:
Fixed.  Procedure type changed to `string` and string returned with added values.  Range check added to ensure we're not overrunning the end of the CSV list.  I went with the maximum range calculation because it allows a check on both ends of the range spectrum as well as ensure we aren't discarding otherwise-valid entries that might look like errors, such as empty strings.  Next commit incoming.

Reply:
Note on the `nRange` validity check.  I used `clamp` to ensure we're not accidentally taking in negative values at the same time as limiting the max value to the maximum possible value of range available based on index and listcount. I considered using the natural return value of the `Add*` functions, but that leaves us with extra loops and, potentially, discarding valid values that look like errors, such as `0`, `0.0`, empty strings, etc.